src/views/Dashboard/Sidepanel/Stats/RegionWiseTable/index.tsx

```diff
@@ -291,6 +362,17 @@ function RegionWiseTable(props: RegionWiseTableProps) {
                     Regions
                 </h3>
                 <div className={styles.actions}>
+                    <MultiSelectInput
```

Comment:
Should this be handled inside the table?
It would be easier.

Reply:
Actually, it would be harder!